src/ngraph/runtime/gpu/cudnn_emitter.cpp

```diff
@@ -943,16 +943,27 @@ size_t runtime::gpu::CUDNNEmitter::build_convolution(const std::string& dtype,
         padding_below, window_movement_strides, window_dilation_strides, mode, data_type);
 
     cudnnConvolutionFwdAlgo_t conv_fwd_algo = CUDNN_CONVOLUTION_FWD_ALGO_IMPLICIT_GEMM;
-    if (find_algo)
+    if (find_algo != algo_search::NONE)
     {
-        CUDNN_SAFE_CALL(cudnnGetConvolutionForwardAlgorithm(*m_ctx->cudnn_handle,
-                                                            tensor_desc_0,
-                                                            filter_desc,
-                                                            conv_desc,
-                                                            tensor_desc_1,
-                                                            CUDNN_CONVOLUTION_FWD_PREFER_FASTEST,
-                                                            /*memoryLimitInBytes=*/0,
-                                                            &conv_fwd_algo));
+        int num_algos;
+        int max_algos = 0;
```

Comment:
we only need 1, so can we set number just to 1?

Reply:
same for others.

Reply:
This max_algos is asking cudnn to try all the algorithms it knows, if we reduce it to 1 I believe it will reduce the number of tested algorithms to 1.

Reply:
I did some test, It does search but only return one result. But I'm not sure if it usable, since I notice there is `status` in perf result, is that means it could be `false`?